dashboard/app/controllers/pd/application/teacher_application_controller.rb

```diff
@@ -18,9 +18,16 @@ def new
       @application = TEACHER_APPLICATION_CLASS.find_by(user: current_user)
       return render :submitted if @application
 
+      options = TEACHER_APPLICATION_CLASS.options.camelize_keys
+
+      school_id = current_user.school_info&.school&.id
+      if school_id
+        options[:school_id] = school_id
+      end
+
       @script_data = {
         props: {
-          options: TEACHER_APPLICATION_CLASS.options.camelize_keys,
+          options: options,
```

Comment:
Let's not pass this down in `options`, which is intentionally static - it's supposed to capture, e.g., all of the dropdown options available, not information about the specific application.  I'd prefer we add a prop below, sibling to `accountEmail` and `userId`.

Reply:
Fixed